cg/store/api/find_business_data.py

```diff
@@ -314,6 +314,20 @@ def get_sample_by_customer_and_name(
             name=sample_name,
         ).first()
 
+    def get_number_of_reads_for_sample_from_sample_lane_metrics(
+        self, sample_internal_id: str
+    ) -> int:
+        """Get number of reads for sample from sample lane sequencing metrics."""
+        total_reads_query: Query = apply_metrics_filter(
+            metrics=self._get_query(table=SampleLaneSequencingMetrics),
+            filter_functions=[
+                SequencingMetricsFilter.FILTER_TOTAL_READ_COUNT_FOR_SAMPLE,
+            ],
+            sample_internal_id=sample_internal_id,
+        )
+        reads_count: Optional[int] = total_reads_query.scalar()
+        return reads_count if reads_count else 0
```

Comment:
Is this used anywhere in the PR? If not I think it should be added when used

Reply:
Apparently not used - I removed it and associated tests. Nothing seems to break because of it. It was not used anywhere in the code